Doc/c-api/structures.rst

```diff
@@ -114,10 +114,20 @@ the definition of all other Python objects.
 
 .. c:type:: PyCFunctionWithKeywords
 
-   Type of the functions used to implement Python callables in C that take
-   keyword arguments: they take three :c:type:`PyObject\*` parameters and return
-   one such value.  See :c:type:`PyCFunction` above for the meaning of the return
-   value.
+   Type of the functions used to implement Python callables in C
+   with signature :const:`METH_VARARGS | METH_KEYWORDS`.
+
+
+.. c:type:: _PyCFunctionFast
```

Comment:
(just a comment) Maybe, we should remove the `_` prefix in 3.9.

Reply:
> Maybe, we should remove the `_` prefix in 3.9.

Probably yes, together with doing the same for the vectorcall API.